tensorboard_plugin_3d/client/babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Overall, I think `.babelrc` is preferred over `babel.config.js` -- could the file be changed?

Reply:
Sure thing! Commit added with this fix.